src/Go/boardAnalysis/goAI.ts

```diff
@@ -26,7 +26,7 @@ let currentAITurn: Promise<Play> | null = null;
 /**
  * Retrieves a move from the current faction in response to the player's move
  */
-export function makeAIMove(boardState: BoardState): Promise<Play> {
+export async function makeAIMove(boardState: BoardState): Promise<Play> {
```

Comment:
I don't think this needs to be, or should be, async. This will be a common theme - just because something returns (or handles) promises, doesn't mean it should be async.